dev/com.ibm.websphere.appserver.features/visibility/public/jakartaeeClient-9.0/io.openliberty.jakartaeeClient-9.0.feature

```diff
@@ -20,6 +20,7 @@ Subsystem-Name: Jakarta EE 9.0 Application Client
   io.openliberty.managedBeans-2.0, \
   io.openliberty.messagingClient-3.0, \
   io.openliberty.persistence-3.0, \
-  io.openliberty.xmlBinding-3.0
+  io.openliberty.xmlBinding-3.0, \
+  io.openliberty.xmlWSClient-3.0
```

Comment:
Is this change related or included by accident?

Reply:
Yes